system_tests/language.py

```diff
@@ -73,25 +73,23 @@ def _check_analyze_entities_result(self, entities):
         # Verify entity 1.
         self.assertEqual(entity1.name, self.NAME1)
         self.assertEqual(entity1.entity_type, EntityType.PERSON)
-        # NOTE: As of Sept. 8, 2016, entity1.salience == 0.81992269
-        self.assertTrue(0.6 < entity1.salience < 1.0)
+        self.assertGreater(entity1.salience, 0.0)
```

Comment:
I think we should mention something about these fluctuating in the docs. Maybe one of those `note` flags?